README.rst

```diff
@@ -64,6 +64,7 @@ A graph object represents a single conceptual graph.
 
 **graph properties**
 
+- An optional `id` property provides an identifier for this graph object
```

Comment:
Missing period.

Reply:
Already accepted PR - feel free to edit the README (I've converted to Markdown since these are fairly simple files as suggested by Travis (@tgig)